babel.config.js

```diff
@@ -15,7 +15,10 @@ module.exports = function (api) {
           ],
           "@babel/preset-typescript",
         ],
-        plugins: [["@babel/plugin-proposal-decorators", { version: "legacy" }]],
+        plugins: [
```

Comment:
So this is giving me some warnings upon starting the app, but its required for class-validator to work with react hook form. If anyone knows the reason to this, I'd be interested in understanding why these warnings are popping up!

Reply:
i think its fine. i remember doing something along these lines for KOH https://github.com/sandboxnu/office-hours/blob/master/packages/app/babel.config.js